src/hitchhiker/tree_cljs.cljc

```diff
@@ -1,4 +1,4 @@
-(ns hitchhiker.tree
+(ns hitchhiker.tree-cljs
```

Comment:
should this be a cljs-file?

Reply:
I have an update to this which unifies the namespaces again.